analysis/model/make_table2_output.R

```diff
@@ -45,15 +45,15 @@ for (cohort in c("prevax","vax","unvax")) {
   # Recalculate total columns --------------------------------------------------
   print('Recalculate total columns')
   
-  table2$total_events_midpoint6_derived <- table2$exposed_events_midpoint6 + table2$unexposed_events_midpoint6
+  table2$total_events_midpoint6_redacted <- table2$exposed_events_midpoint6 + table2$unexposed_events_midpoint6
```

Comment:
This was correct before these changes. It should be `total_events_midpoint6_derived`.

Reply:
Apologies - I see this was a typo in the minutes from our meeting.